packages/user-controllers/src/PreferencesController.ts

```diff
@@ -1,5 +1,9 @@
-import { BaseController, BaseConfig, BaseState } from '../BaseController';
-import { toChecksumHexAddress } from '../util';
+import {
+  BaseController,
+  BaseConfig,
+  BaseState,
+} from '@metamask/base-controller';
+import { toChecksumHexAddress } from '@metamask/controller-utils';
 import { ContactEntry } from './AddressBookController';
```

Comment:
This type is being misused - we shouldn't be using a contact entry type for what it's being used for here. It's not being used as a contract entry.

Once this is resolved, I don't think these two controllers need to be grouped together anymore

Reply:
Both fixed in [90de519](https://github.com/MetaMask/controllers/pull/831/commits/90de5195cf650dd150f4e22135a85a03009bbb12).